src/bp/nlu-core/typings.ts

```diff
@@ -115,15 +115,20 @@ export interface SeededLodashProvider {
 }
 
 export interface Tools {
+  // pre-trained language focused tools
   tokenize_utterances(utterances: string[], languageCode: string, vocab?: string[]): Promise<string[][]>
   vectorize_tokens(tokens: string[], languageCode: string): Promise<number[][]>
   partOfSpeechUtterances(utterances: string[][], languageCode: string): string[][]
   generateSimilarJunkWords(vocabulary: string[], languageCode: string): Promise<string[]>
+  getStopWordsForLang(lang: string): Promise<string[]>
+  duckling: SystemEntityExtractor
+
+  // system info
```

Comment:
Just thinking here its a bit weird that system reside in this tool object

Reply:
Yeah it is lol

Planning on changing this soon